google/cloud/internal/rest_retry_loop_test.cc

```diff
@@ -58,8 +58,8 @@ std::unique_ptr<RetryPolicy> TestRetryPolicy() {
 }
 
 std::unique_ptr<BackoffPolicy> TestBackoffPolicy() {
-  return ExponentialBackoffPolicy(std::chrono::microseconds(1),
```

Comment:
Are these changes related at all?

Reply:
Yeah. The backoff policy returns something in milliseconds, which rounds to 0.

https://github.com/googleapis/google-cloud-cpp/blob/fce4d6864130ec4c0396190ad13ca5537e6f2e1a/google/cloud/internal/backoff_policy.h#L72